frontend/src/features/workspace/components/CreateFormModal/CreateFormModalContent/CreateFormDetailsScreen.tsx

```diff
@@ -79,23 +77,12 @@ export const CreateFormDetailsScreen = (): JSX.Element => {
               <Controller
                 name="responseMode"
                 control={control}
-                render={({ field }) => (
-                  <FormResponseOptions
-                    containsMyInfoFields={containsMyInfoFields}
-                    {...field}
-                  />
-                )}
+                render={({ field }) => <FormResponseOptions {...field} />}
```

Comment:
Might need check if we can remove `containsMyInfoFields` from the wizard completely.